mmdet/apis/ote/apis/detection/openvino_task.py

```diff
@@ -192,20 +133,18 @@ class OpenVINODetectionTask(IInferenceTask, IEvaluationTask, IOptimizationTask):
     def __init__(self, task_environment: TaskEnvironment):
         self.task_environment = task_environment
         self.model = self.task_environment.model
-        self.confidence_threshold: float = 0.0
         self.inferencer = self.load_inferencer()
 
     @property
     def hparams(self):
         return self.task_environment.get_hyper_parameters(OTEDetectionConfig)
 
-    def load_inferencer(self) -> OpenVINODetectionInferencer:
+    def load_inferencer(self):
         labels = self.task_environment.label_schema.get_labels(include_empty=False)
-        self.confidence_threshold = np.frombuffer(self.model.get_data("confidence_threshold"), dtype=np.float32)[0]
```

Comment:
This logic is needed. Confidence threshold is either stored in the model itself, or obtained from the user configuration.